app/components/GearListCard/index.js

```diff
@@ -0,0 +1,45 @@
+/**
+ *
+ * GearListCard
+ *
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+
+import GrayImageOverlay from 'components/shared/GrayImageOverlay';
+
+const ConstrainedImgBySize = styled.img`
+  width: 100%;
+  height: 8rem;
```

Comment:
Why rem? I trust you here just curious

Reply:
It seems to be the trend/safer to use relative units where possible rather than strict pixel-based sizes. From what I've read anyway.